third_party/java/arcs/build_defs/internal/kotlin.bzl

```diff
@@ -160,6 +160,16 @@ def arcs_kt_native_library(**kwargs):
     kwargs["kotlincopts"] = merge_lists(kotlincopts, COMMON_KOTLINC_OPTS)
     kt_native_library(**kwargs)
 
+def arcs_kt_native_binary(**kwargs):
```

Comment:
Why is this added here, but deleted below?

Reply:
This was added here because the kt_native rules require that the "final" wasm artifact be a kt_native_binary. This macro adds kotlincopts to the regular native_binary invokation. 
Below, the deleted portion was a native_library macro.

Reply:
After talking to Martin, this change is no longer relevant.